src/electron/frontend/core/components/pages/guided-mode/setup/Preform.js

```diff
@@ -57,15 +63,31 @@ const questions = {
         },
     },
 
-    timezone: {
-        ...timezoneSchema,
-        title: "What timezone is your data in?",
-        required: true,
+    file_format: {
+        type: "string",
+        enum: ["hdf5", "zarr"],
+        enumLabels: {
+            hdf5: "HDF5",
+            zarr: "Zarr",
+        },
+        strict: true,
+        title: "What file format would you like to use?",
+        description: "Choose a default file format for your data.",
+        default: "hdf5",
+        ignore: true, // NOTE: This ensures that users can only use the default (HDF5) format
+    },
+
+    backend_configuration: {
+        type: "boolean",
+        title: "Will you customize low-level data storage options?",

```

Comment:
```suggestion
        title: "Would you like to customize low-level data storage options?",

```

To be more consistent with the verbiage of the other questions